loaders/src/glTF/2.0/glTFLoader.ts

```diff
@@ -1111,7 +1115,9 @@ export class GLTFLoader implements IGLTFLoader {
         this._babylonScene._blockEntityCollection = false;
 
         // See https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#skins (second implementation note)
-        babylonSkeleton.overrideMesh = this._rootBabylonMesh;
+        if (this._rootBabylonMesh) {
+            babylonSkeleton.overrideMesh = this._rootBabylonMesh;
+        }
```

Comment:
ditto for this

Reply:
Changed by making it null by default as suggested.